tests/auto/rt_auto.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash --login
+set -eux
+if [ -f "accesstoken.sh" ]; then
+  source ./accesstoken.sh
+  export ghapitoken=$ghapitoken
+else
+  echo "Please create accesstoken.sh (600) with the following content\n"
+  echo "/bin/bash\n"
```

Comment:
`/bin/bash` should be prefixed by `#!`, although entire line is not necessary, since `accesstoken.sh` file will never be executed as a stand alone script, it is only going to be sourced from `rt_auto.sh`

Reply:
Yes good catch, and yes i can remove it. If the tests pass, I'll get it changed for the next version.